src/status_im/ui/screens/navigation.cljs

```diff
@@ -51,20 +50,24 @@
     (apply preload-data! db args)))
 
 (defn navigate-to-cofx [go-to-view-id screen-params {:keys [db]}]
```

Comment:
Why it's named `navigate-to-cofx` instead of just `navigate-to` ? Is there some reason why `navigate-to` function should be exposed as public, opposed to make it just `internal-navigate-to` private helper function in the navigation ns ?

Reply:
For the naming https://github.com/status-im/status-react/blob/f36d5d64cc52ccd6bb06fb8743ae9ed535b21d0d/src/status_im/ui/screens/navigation.cljs#L62 
`navigate-to` was already taken and wasn't playing nice with `merge-fx`, so I  started using the `cofx` version leaving `navigate-to` to use the `cofx` version, so we can slowly move away from it.

Reply:
anyways a bit out of the scope of this PR, we can return to this naming later

Reply:
I agree, let's settle the naming later